docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,89 @@
+Run the workflow using example data
+===================================
+
+In this tutorial, you will run the workflow using example reference data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`setup`. These instructions will install all of the software you need to complete this tutorial and others.
+
+Setup
+-----
+
+1. Activate the ``nextstrain`` conda environment:
+
+   .. code:: text
+
+      conda activate nextstrain
+
+2. Change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+3. Download the example tutorial repository into a new directory ``ncov-tutorial/``:
+
+   .. code:: text
+
+      $ git clone https://github.com/nextstrain/ncov-tutorial
+      Cloning into 'ncov-tutorial'...
+      [...more output...]
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a configuration file provided in the tutorial directory:
+
+.. code:: text
+
+   nextstrain build . --cores 4 --configfile ncov-tutorial/example-data.yaml
```

Comment:
This failed for me at the tree building step with the following error:

```
ERROR: Shell exited 2 when running: iqtree -nt 4 -s results/default-build/masked_filtered-delim.fasta -m GTR -ninit 2 -n 2 -me 0.05 -ninit 10 -n 4 > results/default-build/masked_filtered-delim.iqtree.log
Command output was:
  ERROR: You have specified more threads than CPU cores available
```

I've only allocated 3 of my 4 total CPUs to Docker containers which IQ-TREE detects here. If I run the above command without specifying the number of cores, the CLI will use all cores available to the container:

```
$ nextstrain build . --configfile ncov-tutorial/example-data.yaml
Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 3
[...snip...]
```

And the workflow completes as expected. I think we can safely drop the `--cores` argument and corresponding reference below.

Reply:
Oh that's neat! I didn't know that feature of Nextstrain CLI. Agree on dropping `--cores` for all 3 tutorials then.

Reply:
Which version of Nextstrain CLI and Snakemake are you using? With `nextstrain-cli=3.2.1` and `snakemake=7.3.1` I get an error when dropping `--cores`:

> Error: you need to specify the maximum number of CPU cores to be used at the same time. If you want to use N cores, say --cores N or -cN. For all cores on your system (be sure that this is appropriate) use --cores all. For no parallelization use --cores 1 or -c1.

So, the better solution might be to use `--cores all`.

Reply:
Good point. I'm using the nextstrain.cli 3.2.0 with the Docker image `nextstrain/base:build-20220315T184512Z` which uses Snakemake 5.10.0 (where number of cores isn't required). Dropping `--cores` will break things if we ever [update Snakemake some day](https://github.com/nextstrain/docker-base/pull/23/files#r832562522) or for folks running with the native mode.

Using `--cores all` works for me with the CLI + Docker though, so that's a great solution.

Reply:
Done: 3464dd5d